modules/openxr/openxr_interface.h

```diff
@@ -108,6 +108,15 @@ class OpenXRInterface : public XRInterface {
 	virtual PackedStringArray get_suggested_tracker_names() const override;
 	virtual TrackingStatus get_tracking_status() const override;
 
+	// Eye tracking
+	enum EyeGazeCapabilities {
+		XR_EYE_GAZE_UNAVAILABLE, // The eye gaze interaction is not available,
+		XR_EYE_GAZE_LIMITED, // The eye gaze interaction is available in limited form,
+		XR_EYE_GAZE_SUPPORTED, // The eye gaze interaction is supported
```

Comment:
Let's rename these two to explicitly refer to the interaction:
- `XR_EYE_GAZE_INTERACTION_LIMITED`
- `XR_EYE_GAZE_INTERACTION_SUPPORTED`

Reply:
I'm thinking of changing this all together. The specification language is a bit unclear and misleading.

From what I understand the extra structure we can populate during session creation determines whether gaze interaction is available, not whether its full features.

Basically the extension being available means the runtime supports it, the struct queries whether the device in use actually supports it. So for instance, the Meta XR runtime supports eye tracking so the extension will always be available, but only the Quest Pro actually has an eye tracker so only that will return that the gaze is actually supported.

So the outcome of this should be a simple Yes/No, we can use the eye gaze interaction if the extension is available AND we get true on the supported flag.